helpers.js

```diff
@@ -0,0 +1,47 @@
+const ftp = require("basic-ftp");
```

Comment:
It's pretty clear what kind of functions this file contains, let's refer to that, instead of using generic `helper` name.